components/editor/overview.md

```diff
@@ -55,23 +55,6 @@ The <a href = "https://www.telerik.com/blazor-ui/editor" target="_blank">Blazor
 ![Editor first look](images/editor-overview.png)
 
 
-## Component Reference
-
-You can use the component reference to call its [Methods](#methods), especially when creating [custom tools]({%slug editor-custom-tool%}). This snippet shows how to obtain a reference and its namespace.
-
-
-````CSHTML
-@using Telerik.Blazor.Components 
-
-<TelerikEditor @ref="@TheEditorReference"></TelerikEditor>
-
-@code{
-    TelerikEditor TheEditorReference { get; set; }
-}
-````
-
-
-
 ## Dependencies
```

Comment:
Apart from linking ProseMirror, this section has unclear benefit to the customer. Consider shortening it. We use a lot of JavaScript in lots of places already.